frontend/src/app/projects/models/create-coworking-method/create-coworking-method.component.ts

```diff
@@ -41,11 +41,10 @@ import {
 } from 'src/app/services/settings/git-settings.service';
 import {
   CreateGitModel,
-  GitModel,
+  GetGitModel,
+  GitModelService,
   PatchGitModel,
-  SourceService,
-} from 'src/app/services/source/source.service';
-import { GitModelService } from '../../project-detail/model-overview/model-detail/git-model.service';
+} from '../../project-detail/model-overview/model-detail/git-model.service';
```

Comment:
Rather use absolute imports imo
```suggestion
} from 'src/app/projects/project-detail/model-overview/model-detail/git-model.service';
```

Reply:
Agree. Changed my IDE settings (it used "shortest" before to decide for relative/absolute imports). Just checked, relative imports are widespread in the project. I created an issue for it: https://github.com/DSD-DBS/capella-collab-manager/issues/371